File: scripts/sandbox_echo.ts
import process from "node:process";

let input = "";
process.stdin.setEncoding("utf8");
process.stdin.on("data", (chunk) => {
  input += chunk;
});
process.stdin.on("end", () => {
  const payload = input.trim() ? JSON.parse(input) : {};
  process.stdout.write(
    JSON.stringify({
      runtime: "typescript",
      payload
    })
  );
});

File: scripts/sandbox_echo.py
import json
import sys
import time


def main() -> None:
    raw = sys.stdin.read().strip()
    payload = json.loads(raw) if raw else {}
    if "sleep" in payload:
        time.sleep(payload["sleep"])
    sys.stdout.write(
        json.dumps(
            {
                "runtime": "python",
                "payload": payload,
            }
        )
    )


if __name__ == "__main__":
    main()
